AmplicaLabs/social-app-template
# 32 remove v2 from endpoints

# Purpose
The goal of this PR is to remove the `v1` and `v2` prefixes from endpoints.

Closes #32 

## Solution

Remove all `v1/` and `v2/` prefixes from endpoint naming and calls to those endpoints.


### Change summary
* Removed `v1/` and `v2/` prefixes

## Steps to Verify
1. Run swagger and check to see that no endpoints have `v1/` or `v2/` naming
2. When testing an endpoint, you should not see `v1/` or `v2/` in the Request URL.

## Additional details / screenshot
